{"name":"Get: Failure","fullName":"TestTrackPgRepoSuiteRunner/TrackPgRepoSuit/TestTrackPgRepo_GetFailure","status":"passed","statusDetails":{"message":"","trace":""},"start":1728207674850,"stop":1728207674850,"uuid":"211483b0-83c7-11ef-8b52-be2bab8c44a2","historyId":"bbe4bf5b2dd576c8e1d429bb965ac938","testCaseId":"0f0bc11f456e49f48fc589884033e35d","labels":[{"name":"language","value":"go1.23.0"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"MBP-Pavel-2.netis.cc"},{"name":"thread","value":"TestTrackPgRepoSuiteRunner/TrackPgRepoSuit/TestTrackPgRepo_GetFailure"},{"name":"suite","value":"TrackPgRepoSuit"},{"name":"package","value":"cookdroogers/internal/repo/pg"},{"name":"tag","value":"TrackPgRepo"}],"steps":[{"name":"Failure","status":"passed","start":1728207674850,"stop":1728207674850,"steps":[{"name":"ASSERT: Nil","status":"passed","start":1728207674850,"stop":1728207674850,"parameters":[{"name":"Actual","value":"*models.Track((*models.Track)(nil))"}]},{"name":"ASSERT: Error Is","status":"passed","start":1728207674850,"stop":1728207674850,"parameters":[{"name":"Error","value":"sql: connection is already closed: error in postgres repo"},{"name":"Target","value":"error in postgres repo"}]}]}]}